dev/src/index.ts

```diff
@@ -207,9 +207,26 @@ const MAX_CONCURRENT_REQUESTS_PER_CLIENT = 100;
 
 /**
  * Update data (for use with [update]{@link DocumentReference#update})
- * that contains paths (e.g. 'foo' or 'foo.baz') mapped to values. Fields that
+ * that contains paths mapped to values. Fields that contain dots
```

Comment:
delete extra "contain dots" in next line.

Reply:
Ugh. Thanks.